### PR TITLE
Issue 97 Play user-configured alarm sound when alarm is fired

### PR DESCRIPTION
**Pull request by richardkriesman**

### CODE REVIEW

**Comment by ChaoticWeg** [comment]
Will review shortly.

File: src/screens/HomeScreen.tsx [hunk 1]
```diff
@@ -179,21 +184,27 @@ export class HomeScreen extends UIScreen<HomeScreenProps, HomeScreenState> {
...
                 })
                 .then(() => { // audio mode has been set, play the alarm
-                    const sound = new Audio.Sound();
-                    this.setState({
-                        activeSound: sound
-                    }, () => { // active sound has been updated, load and play the alarm
-                        sound.loadAsync(require("../../assets/audio/MusicBox.mp3"))
-                            .then(() => {
-                                return sound.setStatusAsync({
+                    getAlarmSound(this.state.activeSchedule.audio)
```
Comment (ChaoticWeg) [inline src/screens/HomeScreen.tsx:250]: `then` in an async function: necessary? Just a question, will not impact approval

Comment (ChaoticWeg) [reply src/screens/HomeScreen.tsx:250]: I mean rather than `await`

**Review by ChaoticWeg — DISMISSED** [review]
answer my question coward

File: src/screens/HomeScreen.tsx (hunk 1 above)
Comment (richardkriesman) [reply src/screens/HomeScreen.tsx:250]: oh duh

Comment (richardkriesman) [reply src/screens/HomeScreen.tsx:250]: Wait I misread the code. This function isn't async. You can't nicely use `setState` in async functions, hence the promises.

**Review by ChaoticWeg — APPROVED** [review]
👀👀👀👀👀👀👀👀👀👀👀

**Comment by ChaoticWeg** [comment]
travis you traitor pass the build so we can close this

**Pull request merged**